Assets/MRTK/SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -429,20 +452,31 @@ private void UpdateUVMapping()
                     }
                 }
 
-                for (int i = 0; i < uvs.Count; ++i)
+                if (scrollOnly)
                 {
-                    uvs[i] = new Vector2(xLimited ? uvs[i].x : uvs[i].x - uvDelta.x, yLimited ? uvs[i].y : uvs[i].y - uvDelta.y);
+                    for (int i = 0; i < uvs.Count; ++i)
+                    {
+                        uvs[i] = new Vector2(xLimited ? uvs[i].x : uvs[i].x + uvDeltas[i].x, yLimited ? uvs[i].y : uvs[i].y + uvDeltas[i].y);
+                    }
+                }
+                else
+                {
+                    if (!xLimited && !yLimited)
+                    {
+                        for (int i = 0; i < uvs.Count; ++i)
+                        {
+                            uvs[i] += uvDeltas[i];
+                        }
```

Comment:
This case looks identical to lines 475-477. Is there any way to rework these `if`s to dedupe?

Reply:
Yeah did some work but I don't think there's a good way to dedupe this without reducing the code's clarity unfortunately.

Was able to remove the duplicated calls though so see that in the followup PR #9701